WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -95,36 +98,14 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
         self.cardReaderConnectionViewModel = CardReaderConnectionViewModel(cardPresentPayment: cardPresentPaymentService)
         self.orderService = orderService
 
+        self.itemSelectorViewModel = .init(itemProvider: itemProvider)
```

Comment:
nit on L96 & L86: we can probably remove `itemProvider: POSItemProvider` from `PointOfSaleDashboardViewModel` now 😄